pkg/client/clientset/versioned/typed/jenkins.io/v1/pipelineactivity.go

```diff
@@ -32,9 +32,8 @@ type PipelineActivityInterface interface {
 
 // pipelineActivities implements PipelineActivityInterface
 type pipelineActivities struct {
-	client           rest.Interface
-	ns               string
-	returnPatchValue bool
```

Comment:
@jstrachan this bool will disappear on re-generation of the clientset. This file is generated ;-)